packages/apps/docs/src/docs/build/pact/beginner/language-basics.md

```diff
@@ -210,384 +201,425 @@ Here is the same **helloWorld** smart contract with comments included in the mod
-
-|     |                             |                               |
-| --- | --------------------------- | ----------------------------- |
-| !=  | True if X does not equal Y. | (!= "hello" "goodbye") = true |
-| \<  | True if X \< Y.             | (\< 1 3) = true               |
-| \<= | True if X \<= Y.            | (\<= 5.24 2.52) = false       |
-| =   | True if X equals Y.         | (= [1 2 3] [1 2 3]) = true    |
-| >   | True if X > Y.              | (> 1 3) = false               |
-| >=  | True if X >= Y.             | (>= 1 3) = false              |
-
-Along with arithmetic operators and comparison operators, Pact supports boolean, exponential, rounding, and many other common operators. You can learn more about each of these in the
-
-[operator documentation](/reference/functions/operators).
-
-### General Built-in Functions
-
-General functions are responsible for common tasks like manipulating lists, assigning values, checking values, and have many other use cases.
-
-### At
-
-The first function listed on this page is [at](/reference/functions/general#ath3123).
-
-To run the **at** function, type the following command into your terminal and hit enter.
-
-```bash title=" "
-pact > (at 1 [1 2 3])
-2
-```
-
-As you can see, I get a result of 2, which is the 1st index of this list.
-
-You can also search for other values by changing the index. Try updating this to look at index 2 and you should get back a 3.
-
-#### Get a Value from an Object
-
-You can also use **At** to get the value from an object. By specifying the object key, you can return the value of that key from the object.
-
-```bash title=" "
-pact > (at "name" { "type": "cat", "name": "Scratchy", "age": 6 })
-Scratchy
-```
-
-### Bind
-
-**[Bind](/reference/functions/general#bindh3023933)** allows you to map a variable to a value from within an object.
-
-To create a binding, use the keyword **bind** followed by a source object. Follow this object with another object containing a specific value within the source using the := symbol.
-
-```bash title=" "
-pact > (bind { "a": 1, "b": 2 } { "a" := a-value } a-value)
-1
-```
-
-**Example**
-
-Bindings are valuable when you want to bind the values of a table to a variable. Here is a brief example showing how they can be used within a function.
-
-```pact title=" "
-(defun pay (from to)
-    (with-read payments from { "balance":= from-bal }
-    ...code
-)
-```
-
-This example reads a table named **payments** that includes a user **from** that is sending a balance. A binding is used in this case to map the **balance**, which is a column in the table, to the value of **from-bal** that is provided by the user.
+## Built-in functions
+
+After getting familiar with the Pact language syntax, you are ready to start working with some of Pact [built-in functions](/reference/functions).
```

Comment:
```suggestion
After getting familiar with the Pact language syntax, you are ready to start working with some of Pact's [built-in functions](/reference/functions).
```

Reply:
I want to avoid using the possessive here.